pyodide-build/pyodide_build/common.py

```diff
@@ -232,15 +232,14 @@ def get_build_environment_vars() -> dict[str, str]:
     # TODO: Add modifiable configuration file instead.
     # (https://github.com/pyodide/pyodide/pull/3737/files#r1161247201)
     env.update({key: os.environ[key] for key in BUILD_VARS if key in os.environ})
-
     env["PYODIDE"] = "1"
 
-    if "PYODIDE_JOBS" in os.environ:
```

Comment:
I think I removed it by mistake while resolving merge conflicts.

Reply:
I'm not really sure what it does but if we wanted to remove it we'd need to discuss it more specifically.

Reply:
When building libraries, `PYODIDE_JOBS` is passed as a commandline parameter of `make`. I'm not sure how much it affects the build performance, but yeah we discuss it later.

Reply:
Oh, now I remember that I removed this because it is redundant.

`env.update({key: os.environ[key] for key in BUILD_VARS if key in os.environ})` will overwrite every variable that is defined in the host environment so we don't need to do it again for PYODIDE_JOBS.